src/scripts/docs.ts

```diff
@@ -19,6 +30,94 @@ const writeFile = promisify(fs.writeFile)
 if (module.parent === null) update()
 
 async function update(): Promise<void> {
+  const readmePath = path.join(__dirname, '..', '..', 'README.md')
+  let readme = await readFile(readmePath, 'utf8')
+
+  readme = await themeDocs(readme)
+  readme = await extsDocs(readme)
+  readme = await apiDocs(readme)
+
+  await writeFile(readmePath, readme)
```

Comment:
Stylistic suggestion: might be idiomatic promise-wise to have something like:
```JavaScript
  const readme = readFile(readmePath, 'utf8')
                 .then(themeDocs)
                 .then(extsDocs)
                 .then(apiDocs)
                 .catch(/*handle the error*/)
```
perhaps renaming the function names with a `generate` prefix?

If the three doc generation functions don't need to run sequentially(?), `Promise.all` might be a candidate here.

Reply:
Thanks David, good suggestions (that reassignment definitely didn't feel right but I didn't stop to think how it could be done better :slightly_smiling_face: ).